site/snippets/copyright.php

```diff
@@ -1,12 +1,12 @@
-<div>
+<div class="text_with_link">
     <p>
         Copyright<br>
-        © Copyrigt 2023 <?php snippet('co-structAG') ?>. Alle Rechte vorbehalten. Sämtliches Material auf dieser Webseite ist Eigentum von <?php snippet('co-structAG') ?> und darf nur mit Erlaubnis der Eigentümer verwendet werden.<br>
+        © Copyrigt <?php echo date("Y"); ?> <?php snippet('co-structAG') ?>. Alle Rechte vorbehalten. Sämtliches Material auf dieser Webseite ist Eigentum von <?php snippet('co-structAG') ?> und darf nur mit Erlaubnis der Eigentümer verwendet werden.<br>
```

Comment:
echo is not allowed in kirby :)
```suggestion
        © Copyrigt <?= date("Y"); ?> <?php snippet('co-structAG') ?>. Alle Rechte vorbehalten. Sämtliches Material auf dieser Webseite ist Eigentum von <?php snippet('co-structAG') ?> und darf nur mit Erlaubnis der Eigentümer verwendet werden.<br>
```